shared/actions/chat.js

```diff
@@ -1054,15 +1054,18 @@ function * _ensureValidSelectedChat (onlyIfNoSelection: boolean, forceSelectOnMo
 
 const followingSelector = (state: TypedState) => state.config.following
 
-let _loadedInboxOnce = false
 function * _loadInboxMaybeOnce (action: LoadInbox): SagaGenerator<any, any> {
-  if (!_loadedInboxOnce || action.payload.force) {
-    _loadedInboxOnce = true
+  console.log('Load inbox? complete:', _inboxComplete, 'error:', _inboxError, 'force:', action.payload.force)
+  if (!_inboxComplete || _inboxError || action.payload.force) {
```

Comment:
IDK if the service fell over (maybe crashed?) then complete wouldn't get called.
If there was a bug that prevented rpc call, etc etc.